unsafe/src/main/java/org/apache/spark/unsafe/bitset/BitSetMethods.java

```diff
@@ -71,8 +71,11 @@ public static boolean isSet(Object baseObject, long baseOffset, int index) {
    * Returns {@code true} if any bit is set.
    */
   public static boolean anySet(Object baseObject, long baseOffset, long bitSetWidthInBytes) {
```

Comment:
Here, bitSetWidthInBytes should be changed to bitSetWidthInWords and the rest of the code should be updated accordingly.